_posts/2023-05-24-windows-setup.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "ARCS Lab Windows PC Setup"
+tags: ["list", "of", "tags"]
+author: "Anjali Nuggehalli"
+"Here's how we set up a Windows PC to use with various applications"
```

Comment:
This sentence needs to be below the header meta data (underneath the `---`).

And add a blank line in between.